api/openapi-spec/version.json

```diff
@@ -0,0 +1,77 @@
+{
+  "swagger": "2.0",
+  "info": {
+   "title": "Kubernetes /version",
+   "version": "unversioned"
+  },
+  "paths": {
+   "/version/": {
+    "get": {
+     "description": "get the code version",
+     "consumes": [
+      "application/json"
```

Comment:
maybe this is me being ignorant about OpenAPI, but it's a little weird that the /version endpoint consumes something :)